pkg/proxy/endpoints.go

```diff
@@ -282,13 +282,13 @@ func getLastChangeTriggerTime(annotations map[string]string) time.Time {
 	return val
 }
 
-// endpointsChange contains all changes to endpoints that happened since proxy
+// endpointChange contains all changes to endpoints that happened since proxy
```

Comment:
Should this be "endpointChanges" - is *is* plural, and the new name makes it sound singular.

Reply:
"change" rather than "changes" is correct IMO; it describes how the endpoint IPs for a particular service have changed since the last sync. Regardless of how many events there were and how many EndpointSlice objects are involved and how many endpoint IPs they have, the object here is only describing a single change: the change between what we synced last time, and what we should sync next time.

Arguably it could/should stay as "endpoints", not "endpoint", because it encompasses all of the endpoint IPs associated with a particular service...

Reply:
Yeah, SOMETHING is plural here - I don't much care which half, but `endpointChanges` makes most sense to me